src/components/Header/Header.tsx

```diff
@@ -1,26 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useTheme } from '../ThemeContext';
 import { Logo } from '../../assets/icons/Logo';
 
 export const Header: React.FC = (): React.ReactElement => {
   const { theme, setTheme } = useTheme();
   const isActive = (currentTheme: string): boolean => theme === currentTheme;
-
+ const [isDesktop, setIsDesktop] = useState(window.innerWidth >= 768);
+   useEffect(() => {
+     const handleResize = () => {
+       setIsDesktop(window.innerWidth >= 768);
+     };
+ 
+     window.addEventListener('resize', handleResize);
+     return () => window.removeEventListener('resize', handleResize);
+   }, []);
```

Comment:
Why do we need this state? Can't we use breakpoints in CSS? https://tailwindcss.com/docs/responsive-design